src/hotkey/hotkey_command.hpp

```diff
@@ -308,24 +309,31 @@ bool is_scope_active(hk_scopes s);
 
 bool has_hotkey_command(const std::string& id);
 
-/**
- * adds a new wml hotkey to the list, but only if there is no hotkey with that id yet on the list.
- * the object that is created here will be deleted in "delete_all_wml_hotkeys()"
- */
-void add_wml_hotkey(const std::string& id, const t_string& description, const config& default_hotkey);
+class wml_hotkey_record
```

Comment:
This could do with documentation, and a better name. "Record" seems almost as generic as "Object" in this instance.

Reply:
What name would you suggest?